src/status_im/subs/wallet/activities.cljs

```diff
@@ -65,12 +64,25 @@
       {:type :address :address address})))
 
 (defn- process-base-activity
-  [{:keys [timestamp sender recipient token-in token-out chain-id-in chain-id-out activity-status]
+  [{:keys [timestamp sender recipient token-in token-out amount-in amount-out symbol-in symbol-out
```

Comment:
@smohamedjavid, I have a question regarding `activity` structure. Are there situations when `symbol-in` is different from `(:symbol token-in)`? Because it looks like redundancy if no.

Reply:
No @vkjr - The `token-in` or `token-out` map will have only the `token-type`, `chain-id` and `address` (token address) keys. Not the symbol of the token.

Reply:
@smohamedjavid, thanks for clarification!